whisper_live/server.py

```diff
@@ -154,7 +155,7 @@ def recv_audio(self,
                 options["model"] = faster_whisper_custom_model_path
             client = ServeClientFasterWhisper(
                 websocket,
-                multilingual=options["multilingual"],
+                multilingual=False,
```

Comment:
We dont need this as well. This can be removed.

Reply:
We can run the FasterWhisper backend without multilingual, this can be removed from this backend. So we can remove these lines as well:
https://github.com/collabora/WhisperLive/blob/60455b1583b8f4521d0b0ccae38c0df372b73789/whisper_live/server.py#L368

https://github.com/collabora/WhisperLive/blob/60455b1583b8f4521d0b0ccae38c0df372b73789/whisper_live/server.py#L157

Which brings me to the client, and client already sends the language with the model so, it doesn't need to worry about sending the multilingual option as well. If you don't mind updating the client i.e. removing the multilingual option from there that would be really appreciated, else we can take a look at it as well.

The TensorRT backend would need it for the tokenizer.